doc/source/security/data-at-rest-encryption.rst

```diff
@@ -1,13 +1,15 @@
 .. _data_at_rest_encryption:
 
 ================================================================================
-Data at Rest Encryption
+Transparent Data Encryption
```

Comment:
I assume that title change is intentional, right?

Reply:
Yes, I changed it to match the series of blog posts.